### PR TITLE
Godot 4.0

### PR DESCRIPTION
**Pull request by Fifut**
Update to godot 4.0

### CODE REVIEW

File: addons/godot_piskel_importer/piskel_importer.gd [hunk 1]
```diff
@@ -1,50 +1,55 @@
...
 
-# We save directly to stex because ImageTexture doesn't work for some reason
-func get_save_extension():
-	return "stex"
+func _get_save_extension():
+	return "tres"
 
-func get_resource_type():
+func _get_resource_type():
 	return "StreamTexture"
```
Comment (Technohacker) [inline addons/godot_piskel_importer/piskel_importer.gd]: Should this be ImageTexture instead?

**Review by Technohacker — COMMENTED** [review]
Thank you for the PR! Just one thing I wanted to check before I merge :)

File: addons/godot_piskel_importer/piskel_importer.gd (hunk 1 above)
Comment (Fifut) [reply addons/godot_piskel_importer/piskel_importer.gd]: Yes, you are right. StreamTexutre no longer exists in Godot 4.0

Comment (Technohacker) [reply addons/godot_piskel_importer/piskel_importer.gd]: Alright I'll commit that now :)
```suggestion
	return "ImageTexture"
```

**Review by Technohacker — APPROVED** [review]

**Comment by Technohacker** [comment]
Thank you again! :D

**Comment by Fifut** [comment]
Thank you too 👍 
Do not forget to update the "About" section.

**Pull request merged**